notebooks/Deploy embed v3 model.ipynb

```diff
@@ -83,8 +83,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# TODO: replace with the cohere embed v3 model packages when ready\n",
-    "cohere_package = \"cohere-english-small-v2-2-5c5864d6b4ee3277bbbca80225b5638f\"\n",
+    "\"\"\"\n",
+    "English: cohere-embed-english-v3-0-894d9aa7482539ea9c1d8c06097a37ed\n",
```

Comment:
where do these identifiers come from? i only ask because of a typo below, we probably should just copy paste

Reply:
Andrew sent them to me - it had the typo in it

Reply:
cc: @andrewvonrosenbach

Reply:
this is copy pasted from the AWS portal. The way to pull these is to go to the public listings, subscribe to the product, go the "configuration" screen and copy the ARN from there. It likely just truncated due to the length of the package name

![image](https://github.com/cohere-ai/cohere-aws/assets/9967353/7079f7ba-7316-43d7-97b6-1a4fa5703f0b)

Reply:
awesome - I just double checked all the configurations and they look good